src/types.rs

```diff
@@ -218,6 +224,20 @@ pub enum CommercialLengthParseError {
     InvalidLength(u64),
 }
 
+/// Custom reward redemption statuses: UNFULFILLED, FULFILLED or CANCELED
+#[derive(PartialEq, serde::Serialize, serde::Deserialize, Clone, Debug)]
+pub enum CustomRewardRedemptionStatus {
+    /// Unfulfilled reward - the user has claimed it but it is still pending.
+    #[serde(rename = "UNFULFILLED")]
+    Unfulfilled,
+    /// Fulfilled reward - the user has claimed it and the reward has been granted.
+    #[serde(rename = "FULFILLED")]
+    Fulfilled,
+    /// Cancelled reward - the reward has been cancelled before fulfillment, and any spent points have been refunded.
+    #[serde(rename = "CANCELED")]
+    Cancelled,
```

Comment:
is it one or two L's ?

Reply:
Huh, turns out that Americans spell this with one l. I thought it was a typo in twitch's part. I'll change it to Canceled for consistency.